server/server.js

```diff
@@ -45,24 +52,28 @@ app.prepare()
         server.use(bodyParser.json());
         // morgan middleware
         server.use(morgan('dev'));
-        server.use(cookieParser());
+        server.use(cookieParser(SESSION_SECRET));
+
         // config express-session
         const sess = {
-            cookie: {},
+            cookie: { maxAge: 1800000 },
```

Comment:
create `MAX_AGE` constant

Reply:
Should this be inside .env enivronment variable or a constant.js file

Reply:
Should this be inside .env enivronment variable or a constant.js file